docs/usage/configuration-options.md

```diff
@@ -1190,6 +1190,23 @@ A preset alternative to the above is:
 }
 ```
 
+To match specific ports you have to add a protocol to `matchHost`:
+
+```json
+{
+  "hostRules": [
+    {
+      "matchHost": "https://domain.com:9118",
+      "enabled": false
+    }
+  ]
+}
+```
+
+<!-- prettier-ignore -->
+!!! warning
+    Using `matchHost` without a protocol behaves the same as if you had set no `matchHost` configuration.
```

Comment:
I'm wondering if we should copy/paste this admonition into the `matchHost` config option docs?

Reply:
I think that would be too verbose